homeassistant/__main__.py

```diff
@@ -356,7 +356,8 @@ def try_to_restart() -> None:
 
 def main() -> int:
     """Start Home Assistant."""
-    monkey_patch_asyncio()
+    if not sys.version_info >= (3, 5):
```

Comment:
Bug fix has only been included with Python 3.5.3 and 3.6 so we should check at least 3.5.3 here.

https://bugs.python.org/issue26617#msg277825